backend/amd64/emit.mlp

```diff
@@ -1730,7 +1751,6 @@ let emit_instr fallthrough i =
   | Lraise k ->
       begin match k with
       | Lambda.Raise_regular ->
-          I.mov (int 0) (domain_field Domainstate.Domain_backtrace_pos);
```

Comment:
I don't understand why this line was removed, it is absent upstream too so it is probably correct but it was introduced by https://github.com/ocaml-flambda/ocaml-jst/commit/c06038a0eef0eb7f8fda789fb201fefa4df28180 to preserve existing semantics.